components/eamxx/cime_config/testdefs/testmods_dirs/scream/ntasks/16/shell_commands

```diff
@@ -0,0 +1,2 @@
+./xmlchange NTASKS=16 
+
```

Comment:
@jgfouca @bartgol Is this the best way to change the ntasks for CIME tests in mappy? DP test has 25 elements, so the default mpi tasks on mappy (64) was too large.

Reply:
Yes, this is ok. I am not sure, however, how an xmlchange of ntasks plays with ERP/PEM tests. Maybe @jgfouca knows. I think it's ok, but there may be some gotchas for those tests.

Reply:
NVM, this is outdated by the last commit.